src/lib/schema/onboarding.ts

```diff
@@ -8,6 +8,6 @@ export const OnboardingRequest = z.object({
 
 export const OnboardingResponse = z.object({
   domain: z.string().min(1, { message: "Domain is required" }),
+  isDomainDuplicate: z.boolean(),
+  message: z.string().optional(),
```

Comment:
Message is already part of Error schema. Do we still need it here? When the API response code is "not ok" I think we can assume we have an error response.